### PR TITLE
Update code for Botservice

### PR DESCRIPTION
**Pull request by Youri970410**
## Description

update code for Botservice

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/BotService/test/Az.BotService-TestResults.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (msJinLei) [inline src/BotService/test/Az.BotService-TestResults.xml:1]: Please do not commit this file

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please add the description item to the changelog.md

**Comment by wyunchi-ms** [comment]
Please add change log in `ChangeLog.md`.

File: src/BotService/test/Az.BotService-TestResults.xml (hunk 1 above)
Comment (Youri970410) [reply src/BotService/test/Az.BotService-TestResults.xml:1]: Got it,thanks

**Review by msJinLei — DISMISSED** [review]

File: src/BotService/Changelog.md [hunk 2]
```diff
@@ -22,3 +22,6 @@
 ## Version 0.1.0
 * First preview release for module Az.BotService
 
+
+## Version 0.1.1
```
Comment (msJinLei) [inline src/BotService/Changelog.md]: new item should be under upcoming release

**Review by msJinLei — APPROVED** [review]

**Pull request merged**